src/components/HomePage/Headline.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react'
+import styled from '@emotion/styled'
+
+const StyledHeadline = styled.h1`
+  position: relative;
+  top: 5%;
+  font-family: GH Guardian Headline;
+  font-size: 75px;
+  line-height: 101.6%;
+  color: #ffffff;
+  margin-bottom: 16px;
+  /* Tablet/Landscape Mobile Phone */
+  @media (min-width: 650px) and (max-width: 812px) {
+    font-size: 48px;
+  }
+  /* Mobile Phones */
+  @media (min-width: 320px) and (max-width: 649px) {
+    font-size: 36px;
+    margin-bottom: 8px;
+  }
```

Comment:
Should we use Source's sizes here?

If we're venturing in the bold new world of CSS, `clamp()` is [quite interesting][clamp]!

[clamp]: https://css-tricks.com/min-max-and-clamp-are-css-magic/

Reply:
We should definitely be using Source here but given that refactoring and normalising the whole project is an upcoming task, it might make sense to keep it as part of that process instead of trying to fit it in here.

In fact, how and where we are going to put shared styles is probably an important consideration we need to determine and document.